### PR TITLE
new vega lite version

### PR DESCRIPTION
**Pull request by wd60622**
Support for newer vega lite version. 

Had trouble rendering the boxplot mark with the current master branch

### CODE REVIEW

**Review by Conengmo — APPROVED** [review]

**Comment by wdeanHPA** [comment]
The any adjustments to the failing tests from my end? seem to be different than my changes.

File: tests/vegalite_data/vegalite_vlater.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (wd60622) [inline tests/vegalite_data/vegalite_vlater.json:1]: add the schema while testing

**Comment by wd60622** [comment]
Adding v5 after noting the [Altair PR](https://github.com/altair-viz/altair/pull/2528/files).

Restructured the tests to load in the data from JSON. Since the major_version is being tested, switched to public property

**Comment by Conengmo** [comment]
Thank you for your effort @wd60622!

**Pull request merged**